bash2048.sh

```diff
@@ -262,6 +264,70 @@ function reload_game {
   index_max=board_size-1
 }
 
+# print the content of the highscores file
+function show_highscores {
+    output="\nHighscore Name Date\n\n"
+    while read line; do
+        output="${output}${line}\n"
+    done < "${highscore_file}"
+    printf "${output}" | column -t
+}
+
+# check if a score is a high score (in top 10 highest scores)
+# input:
+#         $1: score value
+# output:
+#         0 if true
+#         1 if false
+function is_high_score(){
+    least_score=$(cut -f 1| sort -rn| tail -n 1)
```

Comment:
this does nothing, it does not access the highscore file and also cut needs a delimiter here, otherwise it defaults to tab character and returns entire line

```suggestion
    least_score=$(cat $highscore_file | cut -f 1 -d\  | sort -rn| tail -n 1)
```

Reply:
You're completely right, I must've forgotten the cat at the beginning.